pb/lmsysmd/model/v1/model.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package lmsysmd.model.v1;
+
+import "buf/validate/validate.proto";
+import "google/type/date.proto";
+
+service ModelService {
+  rpc CreateModel(CreateModelRequest) returns (CreateModelResponse) {
+    option idempotency_level = NO_SIDE_EFFECTS; 
+  }
+  rpc GetModel(GetModelRequest) returns (GetModelResponse) {
```

Comment:
I don't think we need to make CRUD endpoints for this if there's no immediate use-case. Also we are reading the google sheet within one endpoint and using a transaction to insert rows, how and when will this model service be used?